contracts/compliance/modular/modules/AbstractModule.sol

```diff
@@ -113,4 +116,23 @@ abstract contract AbstractModule is IModule {
         return _complianceBound[_compliance];
     }
 
```

Comment:
I added these functions to AbstractModule as we'll use them in other modules (ExchangeLimits, MaxBalance, etc.)